gpflow/likelihoods.py

```diff
@@ -203,8 +202,8 @@ def variational_expectations(self, Fmu, Fvar, Y):
 
 
 class StudentT(Likelihood):
-    def __init__(self, deg_free=3.0):
-        Likelihood.__init__(self)
+    def __init__(self, deg_free=3.0, **kwargs):
```

Comment:
"deg_free" sounds pretty terrible... any better idea for how to call it ?

Reply:
degrees_of_freedom, d_o_f, dof??

Reply:
Scipy & numpy just call it `df`... would you prefer `dof`?